autora_closed_loop_demo/researcher_hub/autora_workflow.py

```diff
@@ -21,39 +26,45 @@
 
 from trial_sequence import trial_sequences
 from stimulus_sequence import stimulus_sequence
+from utils import PolynomialRegressor, update_html_script
+from methods import d_prime
+from IPython.display import display
+import os
 
-# *** Set up variables *** #
-# independent variable is coherence in percent (0 - 100)
-# dependent variable is rt in ms (0 - 10000)
-variables = VariableCollection(
-    independent_variables=[
-        Variable(name="S1", allowed_values=np.linspace(1, 100, 100)),
-        Variable(name="S2", allowed_values=np.linspace(1, 100, 100)),
-        ],
-    dependent_variables=[Variable(name="rt", value_range=(0, 10000))])
 
-# *** State *** #
-# With the variables, we can set up a state. The state object represents the state of our
-# closed loop experiment.
+def psudo_experiment_runner():
+    # load a csv file called experiment_data.csv
+    if not os.path.exists("myexperiment.csv"):
+        raise FileNotFoundError("myexperiment.csv not found")
+
+    raw_data = pd.read_csv("myexperiment.csv")
+    return raw_data.to_dict(orient="records")
 
 
-state = StandardState(
-    variables=variables,
-)
+def run_experiment_once():
+    experiment_seq = trial_sequences(
```

Comment:
this and the line below should maybe be the coherence ratios and motion directions used in the actual experiment we're referencing, so:
coherence_ratios=[0, 5, 10, 20, 50]
motion_direction=[70, 160, 250, 340]

Reply:
good point